book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accesor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
```

Comment:
- [ ] Kindly initialize `@rentals` and assign an empty array to it.

  ```
  rentals = [ ]
  ```

Reply:
- Well resolved @Ibtisam34